accurics-aws-tf-sample.tf

```diff
@@ -1,16 +1,16 @@
 provider "aws" {
   access_key = "AKIARWXIFOVSRLGCGYSV" //nitish@accurics.com
   secret_key = "vyRSL5Rn8s6Bd6x9EYmk1ZD7TiEhTTZ/KoYSDgUv"
-  region = "us-west-2" //Canada
+  region     = "us-west-2" //Canada
 }
 
 # Create a VPC to launch our instances into
 resource "aws_vpc" "accurics-test-vpc1" {
   cidr_block = "10.0.0.0/16"
   tags = {
-    Name = format("%s-vpc1", var.acqaPrefix)
+    Name         = format("%s-vpc1", var.acqaPrefix)
     ACQAResource = "true"
-    Owner = "Accurics"
+    Owner        = "Accurics"
   }
 }
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: MySQL (TCP:3306) is accessible by a CIDR block range**
 
**Vulnerability**
'MySQL' (TCP:3306) is accessible by a CIDR block range
 
**Remediation**
Limit the access scope for 'MySQL' to only allow access in internal networks and limited scope. If public interface exists, remove it and limit the access scope within the VPC only to applications or instances that requires access. Amazon Reference: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/authorizing-access-to-an-instance.html. As a further protection step, Use 'Tamper Protection' (Full Protection mode) and region lock to prevent unauthorized changes to network access settings, refer to /360003963194#Set_an_AWS_Security_Group_to_Full_Protection for more details.

source: accurics-app